Package.swift

```diff
@@ -66,6 +67,12 @@ let package = Package(
         .package(url: "https://github.com/tuist/swift-openapi-urlsession", branch: "swift-tools-version"),
     ],
     targets: [
+        .executableTarget(
+            name: "tuistbenchmark",
```

Comment:
Minor: the previous name of the executable was `tuistbench`, either name is fine - the docs would need updating if we'd like to use the new name `tuistbenchmark`.

```suggestion
            name: "tuistbench",
```